app/charts/shared/axis-width-band.tsx

```diff
@@ -77,7 +95,37 @@ export const AxisWidthBandDomain = () => {
   const mkAxisDomain = (
     g: Selection<SVGGElement, unknown, null, undefined>
   ) => {
-    g.call(axisBottom(xScale).tickSizeOuter(0));
+    g.selectAll<SVGGElement, null>(".content")
+      .data([null])
+      .join(
+        (enter) =>
+          enter
+            .append("g")
+            .attr("class", "content")
+            .attr(
+              "transform",
+              `translate(${margins.left}, ${chartHeight + margins.top})`
+            )
+            .call((g) =>
+              g
+                .transition()
+                .duration(TRANSITION_DURATION)
+                .call(axisBottom(xScale).tickSizeOuter(0))
+            ),
+        (update) =>
+          update.call((g) =>
+            g
+              .transition()
+              .duration(TRANSITION_DURATION)
+              .attr(
+                "transform",
+                `translate(${margins.left}, ${chartHeight + margins.top})`
+              )
+              .call(axisBottom(xScale).tickSizeOuter(0))
+          ),
+        (exit) => exit.transition().duration(TRANSITION_DURATION).remove()
+      );
+
```

Comment:
Nit: Seems like we are repeating ourselves here for each axis, could we refactor ?